<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>494</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>450</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>833</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="hsbuttons">
     <item row="0" column="0">
      <widget class="QPushButton" name="TrackingSetUp">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Tracking Set Up</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="RunTherapy">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Therapy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="RightCal">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Right Cal</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="LeftCal">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Left Cal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="RunTest">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Testing</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Quit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>415</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="CameraLayout">
     <item>
      <layout class="QGridLayout" name="RightEyeControls">
       <item row="4" column="1" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="R_CloseCam">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Close Cameas</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="R_RadiusSlider">
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>50</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="R_RadiusLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Radius&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="R_ThresholdSlider">
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>50</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="R_ThresholdLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Thresh&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="R_VertSpace1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLCDNumber" name="R_ThresholdDisplay">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="R_VertSpace2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLCDNumber" name="R_RadiusDisplay">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="Displays">
       <item>
        <widget class="QLabel" name="RightEyeDisplay">
         <property name="minimumSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RightEye&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="CameraLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;▲ Right Eye       Left Eye▼&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="LeftEyeDisplay">
         <property name="minimumSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LeftEye&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="LeftEyeControls">
       <item row="4" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="L_CloseCam">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Close Cameras</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSlider" name="L_RadiusSlider">
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>50</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QSlider" name="L_ThresholdSlider">
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>50</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="L_VertSpace1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="L_VertSpace2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="L_ThresholdLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Thresh&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="L_RadiusLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Radius</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLCDNumber" name="L_RadiusDisplay">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLCDNumber" name="L_ThresholdDisplay">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>801</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Calibration_Buttons">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="leftMargin">
      <number>250</number>
     </property>
     <item>
      <widget class="QLabel" name="Degree_Selection_Label">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Please select calibration amount&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Degree_1_Button">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>1 Degree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Degree_3_Button">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>3 Degrees</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Degree_5_Button">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>5 Degrees</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
